hibernate-core/src/main/java/org/hibernate/annotations/GeneratorType.java

```diff
@@ -83,8 +67,6 @@
 	 * <li>If {@link GenerationTime#ALWAYS}, the value will be generated before
 	 *     each SQL {@code insert} or {@code update} statement is executed.
 	 * </ul>
-	 * @deprecated use {@link #timing()}
 	 */
-	@Deprecated(since = "6.2")
 	GenerationTime when() default GenerationTime.ALWAYS;
```

Comment:
You deprecated ALWAYS in favor of INSERT_OR_UPDATE.  The default should be INSERT_OR_UPDATE here.

* I disagree with INSERT_OR_UPDATE overall, so :shrug:

Reply:
Yeah but the member is also deprecated :)